src/Microsoft.Azure.SignalR.Common/ServiceConnections/scopes/ServiceConnectionContainerScope.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalR
+{
+    internal class ServiceConnectionContainerScope : AsyncLocalScopeBase<IServiceDiagnosticLogsContext>
+    {
+        public ServiceConnectionContainerScope(IServiceDiagnosticLogsContext props) : base(props)
```

Comment:
>IServiceDiagnosticLogsContext [](start = 47, length = 29)

bool EnableMessageLog is enough?

Reply:
Yes, but we can't simply use a `bool`, otherwise we can't update the flag in `HandshakeAysnc`. We have to use a instance.